pkg/crc/machine/machine.go

```diff
@@ -401,3 +405,16 @@ func unsetMachineLogging() {
 	logging.CloseLogFile()
 	logging.SetupFileHook()
 }
+
+func checkBundleCertsValidity(bundleTime string) error {
```

Comment:
Since this method doesn't have any relation to `machine` can we move it to `bundle` package?

Reply:
Thanks, your comment reminded me that i had to change the function, which i did.. and now i think it belongs here, as its getting the cert expiry date from the vm.